documentation/properties/number_of_disputes.md

```diff
@@ -1,13 +1,11 @@
 ---
 layout:		property
-title:		"number_of_disputes"
+title:		"margin_period_of_risk"
 schemas:	[agreement]
 ---
 
-# number_of_disputes
-
-The **number_of_disputes** is defined in accordance with Article 285(4) of the [CRR][CRR]:
-> If an institution has been involved in more than two margin call disputes on a particular netting set over the immediately preceding two quarters that have lasted longer than the applicable margin period of risk under paragraphs 2 and 3, the institution shall use a margin period of risk that is at least double the period specified in paragraphs 2 and 3 for that netting set for the subsequent two quarters.
+# margin_period_of_risk
 
+The **margin_period_of_risk** attribute allows insitutions to provide their estimate of the margin period of risk (MPOR) for the transactions covered by the agreement (CSA) [CRE 52.50]. As the MPOR is subject to a regualtory floor, it can be left empty.
```

Comment:
Can you also add a note about the units. Is it calendar days, business days?

Reply:
will do

Reply:
I have pushed the amendments. Please have another look to ensure it is clear and typo-free!